packages/vscode/resources/prisma-icon-dark.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="100" height="100" viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What are these for? Seems like an unrelated change.

Reply:
Icons for the Studio window